ui/src/normalizers/dashboardRefresh.js

```diff
@@ -0,0 +1,24 @@
+import {isObject} from 'lodash'
+
+const dashrefresh = (refreshes) => {
+  if (!Array.isArray(refreshes)) {
+    return []
+  }
+
+  const normalized = refreshes.filter(r => {
+    if (!isObject(r)) {
+      return false
+    }
+
+    const {dashboardID, refreshRate} = r
+
+    if (typeof dashboardID !== 'number' || !(!refreshRate || typeof refreshRate === 'number')) {
```

Comment:
this `!(!refreshRate || typeof refreshRate === 'number'))` is melting my brain heh. do you think you could pull

`!refreshRate || typeof refreshRate === 'number'` out into a well named and semantic variable so that the expression becomes

```ts
if (typeof dashboardID !== 'number' || !semanticVariableDescribingThings)
```

Reply:
Sure thing.